warehouse/oidc/services.py

```diff
@@ -287,11 +310,30 @@ def find_publisher(
             publisher = find_publisher_by_issuer(
                 self.db, self.issuer_url, signed_claims, pending=pending
             )
+
+            jwt_token_identifier: str | None = signed_claims.get("jti")
+            # jti is in the __preverified_claims__ set, so if it was present,
+            # it was already checked
+            if pending is False and jwt_token_identifier:
+                if self.token_identifier_exists(jwt_token_identifier):
+                    self.metrics.increment(
+                        "warehouse.oidc.reused_token",
+                        tags=metrics_tags,
+                    )
+                    raise ReusedTokenError("JWT Token already used to mint a token.")
```

Comment:
This error message isn't used anywhere as far as I can tell. Would it be better to pass it through when we catch this exception, or should this just be a `raise ReusedTokenError`?

Reply:
Yeah, I think we can remove this message outright.

Reply:
(Also, per my other comment, this error message isn't strictly true in every case, as the token might not have been minted.)